packages/web/pages/apps/index.tsx

```diff
@@ -47,15 +48,20 @@ export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
 
   const { applications } = apps;
 
+  const t = useTranslation();
+  const { logEvent } = useAmplitudeAnalytics();
+  const { width } = useWindowSize();
+
   useEffect(() => {
     const options = {
       keys: ["title"],
     };
     setFuse(new Fuse(applications, options));
   }, [applications]);
 
-  const t = useTranslation();
-  const { logEvent } = useAmplitudeAnalytics();
+  useEffect(() => {
```

Comment:
Log event is memoized so this should still only run once

Reply:
Actually on mount events should be passed as params to `useAmplitudeAnalytics`:

https://github.com/osmosis-labs/osmosis-frontend/blob/7ffa25d8ff0bd2c738a9a75914e69493392529a0/packages/web/hooks/use-amplitude-analytics.ts#L13